module-2/backend/week_one.md

```diff
@@ -5,28 +5,63 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  * GET - Read  
+  * POST - Create 
+  * PUT - Update
+  * DELETE - Delete
+  * PATCH - Update
 2. What is Sinatra?
+ * A web application framework written in Ruby
 4. What is MVC?
+ * Model-View-Controller, a architecture pattern for developing web applications
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+ * RESTful conventions are used to have an understandable standard amongst developers
 6. What types of variables are accessible in our view templates without explicitly passing them?
+ * instance variables
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+  ```ruby
+  get '/horses' do
+    name = 'Mr. Ed'
+    erb :index, locals: {name: name}
+  end
+  ```  
 9. What's the purpose of ERB?
+ * Add Ruby code in HTML templates
 10. Why do I need a development AND test database?
+ * To separate experimental development and refined testing that will eventually lead to the production database
 11. What's responsive design?
+ * Designing sites to respond to screen sizes
 12. What is CRUD and why is it important?
+ * Create, Read, Update, Delete. The basic functionality for applications interacting with a database
 13. What does HTTP stand for? 
+ * Hypertext transfer protocol
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+ * <%= stuff %>, <% stuff %>
+ * the former inserts the interpeted ruby value, the latter is for direct ruby code (usually for conditionals)
 15. What's an ORM?
+ * object relational mapping. usually an OOP language wrapping around a database language so developers can interact with the database easier
 16. What's the most commonly used ORM?
+ * activerecord
 17. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.

```

Comment:
This question is looking more for the verb and the path combinations so what would the path be to read one restaurant - `/restaurants/:id`